src/components/AccountDrawer/MiniPortfolio/Tokens/index.tsx

```diff
@@ -75,10 +76,12 @@ function TokenRow({ token, quantity, denominatedValue, tokenProjectMarket }: Tok
 
   const navigate = useNavigate()
   const toggleWalletDrawer = useToggleAccountDrawer()
+  const isExplore = useInfoExplorePageEnabled()
```

Comment:
rename all of `isExplore` to `useInfoExplorePageEnabled`, `isExplore` is very generic and doesn't convey what the value actually represents